tests/subsys/audio_module/src/audio_module_test_common.c

```diff
@@ -19,6 +20,7 @@ void test_context_set(struct mod_context *ctx, struct mod_config const *const co
 	memcpy(&ctx->test_string, TEST_STRING, sizeof(TEST_STRING));
 	ctx->test_uint32 = TEST_UINT32;
 	memcpy(&ctx->config, config, sizeof(struct mod_config));
+	reset_fake_fifo_counter();
```

Comment:
Verb preferred last: 
```suggestion
	fake_fifo_counter_reset();
```